jekyll/_docs/npm-continuous-deployment.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: classic-docs
+title: "Continuous deployment with npm"
+short-title: "npm publish from CircleCI"
+description: "How to configure CircleCI to publish packages to npm automatically"
+---
+
+Publishing to npm from CircleCI makes it easy for project collaborators to
+publish new versions of your package in a consistent and predictable way.
+
+1.  Obtain the npm authToken for the account that you wish to use to publish
+    the package.
+
+    You can do that by logging in to npm (`npm login`). This will save the
+    authToken to the `~/.npmrc` file. Look for the following line:
+
+    ```
+    //registry.npmjs.org/:_authToken=00000000-0000-0000-0000-000000000000
+    ```
+
+    In this case, the authToken is `00000000-0000-0000-0000-000000000000`.
+
+2.  Go to your project settings, and set the `NPM_TOKEN` variable to the
```

Comment:
Link "project settings" to https://circleci.com/docs/environment-variables/#setting-environment-variables-for-all-commands-without-adding-them-to-git. Proper Jekyll syntax would be `{{site.baseurl}}environment-variables/#setting-environment-variables-for-all-commands-without-adding-them-to-git`.